packages/embark/src/lib/modules/blockchain/index.js

```diff
@@ -19,14 +20,22 @@ class Blockchain {
       this.blockchainNodes[clientName] = startCb;
     });
 
-    this.events.setCommandHandler("blockchain:node:start", (blockchainConfig, cb) => {
+    this.events.setCommandHandler("blockchain:node:start", (blockchainConfig, node, cb) => {
+      if (typeof node === 'function') {
+        cb = node;
+        node = null;
+      }
       const clientName = blockchainConfig.client;
       // const clientName = this.blockchainConfig.client;
+      if (node && node === constants.blockchain.vm) {
+        this.events.emit("blockchain:started", clientName, node);
```

Comment:
How can we be sure that the VM is **actually** started already?

Instead, shouldn't we create a vm plugin and register it just like geth/parity, and then start it here?? As opposed to adding special cases for `node`.